src/applications/personalization/profile-2/components/EmailAddressNotification.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import recordEvent from 'platform/monitoring/record-event';
+
+const EmailAddressNotification = ({ signInServiceName }) => {
+  let link;
+  let buttonText;
+
+  if (signInServiceName === 'idme') {
+    link = 'https://wallet.id.me/settings';
+    buttonText = 'ID.me';
+  }
+
+  if (signInServiceName === 'dslogon') {
+    link = 'https://myaccess.dmdc.osd.mil/identitymanagement';
+    buttonText = 'DS Logon';
+  }
+
+  if (signInServiceName === 'mhv') {
+    link = 'https://www.myhealth.va.gov';
+    buttonText = 'My HealtheVet';
+  }
+
+  return (
+    <>
+      <p className="vads-u-margin--0">
+        To update the email address you use to sign in, go to the account where
+        you manage your settings and identity information. Any email updates you
+        make there will automatically update on VA.gov.
+      </p>
+      <p className="vads-u-margin-bottom--0">
+        <a
+          href={link}
+          target="_blank"
+          rel="noopener noreferrer"
+          onClick={() => {
+            recordEvent({ event: 'update-sign-in-email-address-clicked' });
```

Comment:
I'm not seeing this as an existing event that we are tracking?

Reply:
Will remove it for now.